README.md

```diff
@@ -179,6 +179,13 @@ Please see [Browsersync documentation](https://browsersync.io/docs) for more det
 This error usually occurs when the watcher server is not running.
 Run `ddev browsersync` to start the server.
 
+### Typo3 No site configuration found / 404 Error
+
+Solution: base url in config.yaml should be always http even if the frontend run over https
```

Comment:
```suggestion
Solution: The base URL (`base` in the TYPO3 site's `config.yaml` (like `config/sites/main/config.yaml` should specify `http` even though the frontend is configured for `https`. This can also be solved by commenting out the default `base` line.
```

Reply:
@rfay I agree , thank you!